flink-streaming-connectors/flink-connector-filesystem/src/main/java/org/apache/flink/streaming/connectors/fs/bucketing/BucketingSink.java

```diff
@@ -161,9 +161,7 @@
 	// These are initialized with some defaults but are meant to be changeable by the user
 
 	/**
-	 * The default maximum size of part files.
-	 * <p>
-	 * By default, {@code 6 X} the default block size.
+	 * The default maximum size of part files (currently {@code 384 MB}).
```

Comment:
Actually this must be based on the old defaults of the HDFS block size. A big file in HDFS was split into smaller of this size and each one was stored on a separate node. If I remember correctly 64MB was the default but they may have changed since then.